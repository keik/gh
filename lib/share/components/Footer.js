// @flow

import React from 'react'
import styled from 'styled-components'

import Logo from './common/atoms/Logo'
import InnerContainer from './common/layouts/InnerContainer'

const Footer = () => (
  <$Container>
    <$NavLinks>
      <li>&copy; 2016 keik</li>
      <li>
        <a href="#">Help</a>
      </li>
    </$NavLinks>
    <Logo href="#">GH</Logo>
    <$NavLinks>
      <li>
        <a href="#">API</a>
      </li>
      <li>
        <a href="#">About</a>
      </li>
    </$NavLinks>
  </$Container>
)

export default Footer

const $Container = styled(InnerContainer)`
  display: flex;
  align-items: center;
  justify-content: space-between;
  padding: 24px 0;
  margin-top: 36px;
  border-top: 1px solid #eee;
  color: #767676;
  font-size: 12px;
`
const $NavLinks = styled.ul`
  list-style: none;
  padding: 0;
  margin: 0;
  li {
    display: inline-block;
    + li {
      margin-left: 10px;
    }
  }
`
