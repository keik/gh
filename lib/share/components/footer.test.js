import test from 'tape'
import { shallow } from 'enzyme'
import React from 'react'

import Footer from './footer'

test('<Footer /> with no props should throw no errors', t => {
  t.doesNotThrow(() => {
    shallow(<Footer />)
  })
  t.end()
})
