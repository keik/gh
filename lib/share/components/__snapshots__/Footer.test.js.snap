// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> snapshot 1`] = `
<div
  className="Footer__Container-yz7m8k-0 itNppg"
>
  <ul
    className="Footer__NavLinks-yz7m8k-1 hrZQRx"
  >
    <li>
      Â© 2016 keik
    </li>
    <li>
      <a
        href="#"
      >
        Help
      </a>
    </li>
  </ul>
  <a
    className="Logo-k763vu-0 eMaEfw"
    href="#"
  >
    GH
  </a>
  <ul
    className="Footer__NavLinks-yz7m8k-1 hrZQRx"
  >
    <li>
      <a
        href="#"
      >
        API
      </a>
    </li>
    <li>
      <a
        href="#"
      >
        About
      </a>
    </li>
  </ul>
</div>
`;
